packages/web-pixels-extension/src/types/PixelEvents/index.ts

```diff
@@ -1490,7 +1584,7 @@ export interface Product {
   /**
    * The ID of the product.
    */
-  id: string;
+  id: string | null;
```

Comment:
I had manually discarded these changes on my PRs since they weren't related to what I was adding. But maybe we should add them now? or in a follow up PR?